src/main/webapp/app/entities/flair-bi/flair-bi-content-header.controller.js

```diff
@@ -883,5 +887,28 @@
             if (containerColor)
                 $('.page-wrapper-full-screen').css('background-color', containerColor)
         }
+
+        function share() {
+            $uibModal
+                .open({
+                    animation: true,
+                    templateUrl:
+                        "app/entities/flair-bi/share-dialog/share-dialog.html",
```

Comment:
rename html and controller to ShareDialogController to SharedLinkDialogController and share-dialog.html to shared-link-dialog.html